src/hooks/protections/received_event.cpp

```diff
@@ -741,6 +741,31 @@ namespace big
 		default: break;
 		}
 
+		if (event_id == 74 && g.protections.ptfx_spam_protection)

```

Comment:
Still a raw event id comparison, please add an event called PTFX2 if it's really different from the other PTFX event.

I'm not sure why there would be 2 different PTFX events, perhaps the original event id in the code base is dated.

Reply:
The current eNetworkEvents with value 74 is named NETWORK_CHECK_EXE_SIZE_EVENT. The current NETWORK_PTFX_EVENT did not get triggered for me in the probably ~2 weeks I had it logging for testing purposes.

Reply:
I'm guessing they removed that one from the enum then, aha, nice catch (I didn't even look at the enum so I could've probably seen that myself).

Reply:
> The current eNetworkEvents with value 74 is named NETWORK_CHECK_EXE_SIZE_EVENT. The current NETWORK_PTFX_EVENT did not get triggered for me in the probably ~2 weeks I had it logging for testing purposes.

So one of the remote events have been removed from what i can tell not sure which right now